src/main/java/com/google/sps/servlets/ChatServlet.java

```diff
@@ -0,0 +1,60 @@
+package com.google.sps.servlets;
```

Comment:
don't include chat servlet

Reply:
This file has now been deleted from this pull request